### DISCUSSION TITLE
When multivariate?

### DISCUSSION THREAD

**Discussion by ghost** [ourownstory/neural_prophet#249] [Feature Requests & Feedback]
Greetings,

I didn't see multivariate feature in the timeline, please add it to top priority. 
Also, is there a way to extend it to multivariate in the mean time?

**Comment by ourownstory** [comment]
Hi @veioxve3r1d23seb2g6b we do have it as a priority. If you would like to advance it's addition, we heartily welcome contributions!

And yes, you can model multivariate timeseries already - you can add each covariate as  lagged regressor:
`m = m.add_lagged_regressor(name='A')`

If you are trying to predict multiple target time series, create a model for each of them.
Thus, you can already fully model multivariate time-series.
Hope this helps.

**Reply by ghost** [reply]
thanks @ourownstory, i think future regressor is what i am looking for, since my data doesn't have lag
